### DISCUSSION TITLE
Creating messages not from platform we know

### DISCUSSION THREAD

**Discussion by zombozo12** [chatwoot/chatwoot#4000] [Q&A] [answered]
Hi everyone,

I found chatwoot out of nowhere while exploring the 'explore' page on Github and think it is a great software. But I wonder, can Chatwoot create a message not from Facebook, Twitter, or any platform we know? Let's say I have hosted Chatwoot on a VPS and I develop a mobile app and an API for handling mobile request and responses. In mobile, it have `access_token` from API as an identifier. And then, in the mobile app we bundle user's fullname and email to send a message to Chatwoot API that already hosted on my VPS as "User Messages". Can Chatwoot do that? Or Chatwoot only limited to platform we know?

Thank you.

**Comment by sojan-official** [comment, marked as answer]
@zombozo12  Please take a look at our API channels.You should be able to build custom conversation interfaces leveraging that 

ref: 
https://www.chatwoot.com/docs/product/channels/api/create-channel 
https://www.chatwoot.com/docs/product/channels/api/client-apis

**Reply by zombozo12** [reply]
Thank you for your reply! and wow thank you, I didn't know to look in the docs actually therefore I ask. That helped me a lot! Once again, thank you!!
